docs/developersguide/testing-practices.rst

```diff
@@ -0,0 +1,72 @@
+Testing practices
+=================
+
+It is a good practice to add test cases to each Invenio module. Tests are
+normally organized in:
+
+1. Unit functional testing: test cases run against the Flask app defined in the
```

Comment:
This sounds a bit confusing (but I missed the end of sprint so maybe it was discussed IRL), but what are `unit functional tests`? Shouldn't there be a separation between `unit` tests (where you test some functions from a module) and `functional` tests (where you test the whole module as a `black box`) ?